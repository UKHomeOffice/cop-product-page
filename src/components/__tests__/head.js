import React from "react"
import renderer from "react-test-renderer"
import Head from "../head";

describe("Head", () => {
  it("renders correctly", () => {
    const data = { title: 'title',  subTitle: 'sub title'}  
    const tree = renderer
      .create(<Head data={data} />)
      .toJSON();
    expect(tree).toMatchSnapshot()
  })

  it("renders correctly with no subtitle", () => {
    const dataObj = { title: 'title',  subTitle: 'sub title', subject: 'How to'}  
    const tree = renderer
      .create(<Head data={dataObj} />)
      .toJSON();
    expect(tree).toMatchSnapshot()
  })

  it("renders correctly with no subtitle", () => {
    const childrenObj = { title: 'title'}
    const tree = renderer
      .create(<Head data={childrenObj} />)
      .toJSON();
    expect(tree).toMatchSnapshot()
  })
})