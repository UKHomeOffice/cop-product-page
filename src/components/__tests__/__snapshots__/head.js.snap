// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head renders correctly 1`] = `null`;

exports[`Head renders correctly with no subtitle 1`] = `null`;

exports[`Head renders correctly with no subtitle 2`] = `null`;
